src/main/java/digital/slovensko/autogram/ui/cli/CliSettings.java

```diff
@@ -0,0 +1,97 @@
+package digital.slovensko.autogram.ui.cli;
+
+import digital.slovensko.autogram.core.UserSettings;
+import digital.slovensko.autogram.core.errors.PDFSignatureLevelIsNotValidException;
+import digital.slovensko.autogram.core.errors.SlotIdIsNotANumberException;
+import digital.slovensko.autogram.core.errors.SourceDoesNotExistException;
+import eu.europa.esig.dss.enumerations.SignatureLevel;
+import org.apache.commons.cli.CommandLine;
+
+import java.io.File;
+
+public class CliSettings extends UserSettings {
+    private String target;
+    private File source;
+    private boolean isForce;
+    private boolean shouldMakeParentDirectories;
+
+    public static CliSettings fromCmd(CommandLine cmd) {
+        var settings = new CliSettings();
```

Comment:
Do velkej miery len copy&paste a vyuziva to nastavovanie settings

Reply:
Toto som si predstavoval trochu inak. Teraz máme UserSettings nad týmto, ktoré majú aj nejaké save a load metódy a môžu mať aj iné parametre. Ja by som radšej spravil interface/abstract class `AutogramSettings` a ten by bol implementovaný v UserSettings aj CLISettings osobitne. A môže ostať, že interface/abstract class `AutogramSettings` by implementovali tie tri nové interfaces, čo si vyrobil.

Reply:
Ano, to bol dalsi krok, ktory som uz vcera nezvladol vecer lebo tie settings su zrele na upratovanie lebo aj nazvy su tam vselijake halabala is/has/should/_Enabled